### PR TITLE
Provide screenshot for paused sync state

### PR DESCRIPTION
**Pull request by Cassie205**
Adding Screenshot for "Paused Sync" state and updating alt image title

### CODE REVIEW

File: cyberduck/bookmarks.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ Example configuration:
 | HTTP URL | [http://sudo.ch/stylesheet.css](http://sudo.ch/stylesheet.css) | *Accessible in the web browser* |
 
 ```{image} _images/HTTP_URL.png
-:alt: Send Command
+:alt: HTTP ULR
```
Comment (AliveDevil) [inline cyberduck/bookmarks.md:145]:
```suggestion
:alt: HTTP URL
```

File: cyberduck/connection.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ The *Automatic Proxy Configuration* using a *Proxy Configuration File* is curren
 Choose *FTP Proxy* and select the checkbox to configure the proxy address.
 
 ```{image} _images/Proxy_Configuration_FTP_and_SFTP.png
-:alt: Send Command
+:alt: Proxy Configuration FTP/ SFTP
```
Comment (AliveDevil) [inline cyberduck/connection.md:139]:
```suggestion
:alt: Proxy Configuration FTP/SFTP
```

**Review by AliveDevil — CHANGES_REQUESTED** [review]

File: cyberduck/transfer.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ Some servers may not support resumable transfers and the file will be reloaded i
 ```
 
 ```{image} _images/Transfers.png
-:alt: Send Command
+:alt: Tranfsers
```
Comment (tatze96) [inline cyberduck/transfer.md:20]: Change "Tranfsers" to " Transfers"

File: protocols/dropbox.md [hunk 4]
```diff
@@ -2,7 +2,7 @@ Dropbox
 ===
 
 ```{image} _images/dropbox.png
-:alt: Send Command
+:alt: Bropbox Drive Icon
```
Comment (tatze96) [inline protocols/dropbox.md:5]: Change "Bropbox" to "Dropbox"

**Review by tatze96 — CHANGES_REQUESTED** [review]

**Review by tatze96 — APPROVED** [review]

**Review by AliveDevil — APPROVED** [review]

**Pull request merged**